packages/babel-plugin-transform-classes/src/spec.js

```diff
@@ -0,0 +1,38 @@
+import VanillaTransformer from "./vanilla";
+import { types as t } from "@babel/core";
+
+export default class SpecClassTransformer extends VanillaTransformer {
+  constructor() {
+    super(...arguments);
+    this.isSpec = true;
+  }
+
+  _getMethodRef(node, isStatic) {
+    return t.memberExpression(
+      isStatic
+        ? this.classRef
+        : t.memberExpression(this.classRef, t.identifier("prototype")),
+      t.identifier(node.key.name),
```

Comment:
This is going to have issues if it's a computed method.

Reply:
also probably `this.classRef` should be cloned